Emby.Server.Implementations/Data/SqliteItemRepository.cs

```diff
@@ -4517,17 +4517,17 @@ private List<string> GetWhereClauses(InternalItemsQuery query, IStatement statem
                 }
             }
 
-            if (query.HasImdbId.HasValue)
+            if (query.HasImdbId.HasValue && query.HasImdbId.Value)
```

Comment:
What about the case where it's false? Shouldn't it be `"ProviderIds not like '%imdb=%'"` then?